src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -346,12 +346,12 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
       }
     }
 
-    const loadBucketLocation = async (googleProject, bucketName) => {
+    const loadBucketLocation = withErrorReporting('Error loading bucket location', async (googleProject, bucketName) => {
```

Comment:
I think withErrorIgnoring is more appropriate, as I believe the code is robust to location being undefined. A console.log might be good though.

Reply:
Note that this is also a general issue we are thinking about with regard to Azure workspaces. For a cloned Azure workspace, the storage container will not be immediately present, so blob storage and details calls will need to be robust to that intermediary state.

Reply:
Switched in https://github.com/DataBiosphere/terra-ui/pull/3699/commits/5142b8299dcea4776848a019d82e393b5795483c. And yeah, I am not convinced that everything in WorkspaceContainer is actually resilient to a missing bucket location, but I don't think this change introduces any new problems there.